styles/colors-silicon-day.less

```diff
@@ -54,6 +54,7 @@
 @agate-gray:        #8a8a8a;
 @agate-light-gray:  #ededed;
 @agate-white:       @silicon-white;
+@agate-noble:		#999999;
```

Comment:
This is a silly thing to comment on, but since other changes are being made, I'll mention it. This uses a space followed by a tab, which positions the value out of alignment with the rest. Use spaces to ensure alignment of these sorts of things.

Mind if we use `@agate-nobel-gray` to improve the recognizability of what this variable will represent?

Reply:
Renamed `@agate-noble` with `@agate-noble-gray` and fixed indentation.